docs/naps/0-nap-process.md

```diff
@@ -0,0 +1,297 @@
+err on the side of asking for more feedback and looking for
+opportunities to compromise.
+
+If the final comment period passes without any substantive objections,
+then the NAP can officially be marked `Accepted`. You should send a
+followup post notifying the thread (celebratory emoji optional but
+encouraged 🎉✨), and then update the NAP by setting its `:Status:`
+to `Accepted`, and its `:Resolution:` header to a link to your
+followup post.
+
+If there *are* substantive objections, then the NAP remains in
+`Draft` state, discussion continues as normal, and it can be
+proposed for acceptance again later once the objections are resolved.
+
+In unusual cases, when no consensus can be reached between core developers,
+the [napari Steering Council] may be asked to decide whether a
+controversial NAP is `Accepted`.
+
+### Maintenance
+
+In general, Standards track NAPs are no longer modified after they have
+reached the Final state, as the code and project documentation are
+considered the ultimate reference for the implemented feature. They may,
+however, be updated under special circumstances.
+
+Process NAPs may be updated over time to reflect changes
+to development practices and other details. The precise process followed in
+these cases will depend on the nature and purpose of the NAP being updated.
+
+## Format and Template
+
+NAPs are UTF-8 encoded text files using the [MyST markdown] format.  Please
+see the {ref}`nap_template` file and the [MyST markdown cheat sheet] for
+more information.  We use [Sphinx] to convert NAPs to HTML for viewing on
+the web [^id4].
+
+### Header Preamble
+
+Each NAP must begin with a header preamble.  The headers
+must appear in the following order.  Headers marked with `*` are
+optional.  All other headers are required.
+
+```
+  :Author: <list of authors' real names and optionally, email addresses>
+  :Status: <Draft | Active | Accepted | Deferred | Rejected |
+           Withdrawn | Final | Superseded>
+  :Type: <Standards Track | Process>
+  :Created: <date created on, in dd-mmm-yyyy format>
+* :Requires: <nap numbers>
+* :napari-Version: <version number>
+* :Replaces: <nap number>
+* :Replaced-By: <nap number>
+* :Resolution: <url>
+```
+
+The Author header lists the names, and optionally the email addresses of
+all the authors of the NAP. The format of the Author header value must be
+
+> Random J. User \<<mailto:address@dom.ain>>
+
+if the email address is included, and just
+
+> Random J. User
+
+if the address is not given.  If there are multiple authors, each should be
+on a separate line.
+
+## Discussion
+
+- <https://github.com/napari/napari/pull/4299>
+
+## References and Footnotes
+
+[^id3]: This historical record is available by the normal git commands
+    for retrieving older revisions, and can also be browsed on
+    [GitHub](https://github.com/napari/napari/tree/main/docs/naps).
+
+[^id4]: The URL for viewing NAPs on the web is
+    <https://napari.org/naps/>
+
```

Comment:
Can you maybe add a section on acknowledgements - maybe call out some of the PEP and SKIP processes that I think we're heavily inspired by. Those links could help us resolve issues if we have failed to anticipate them yet and need to look to others for guidance